src/main/webapp/src/app/common/growpod-page-styles.css

```diff
@@ -9,3 +9,15 @@
 .text-center {
   text-align: center;
 }
+.button {
+  display: flex;
+  margin-left: auto;
```

Comment:
Can you create the same effect via `margin: 0 auto;`. I think that is the more common/recommended way to write it.

Reply:
Done